openassessment/xblock/static/js/spec/lms/oa_listing.js

```diff
@@ -298,45 +298,56 @@ describe("OpenAssessment.CourseItemsListingView", function() {
 
 
     describe('esg enchanced grade flag', function() {
-        it('is enabled', () => {
+        describe('is enabled', function() {
             var data = {
                 CONTEXT: {
                     ENHANCED_STAFF_GRADER: true,
                     ORA_GRADING_MICROFRONTEND_URL: 'some_url'
                 }
             };
 
-            view = createCourseItemsListingView('oa_listing_view.html', data);
-            // Add test data
-            view.oraData = testData;
-            expect(view.data).toEqual(data);
+            beforeEach(function() {
+                view = createCourseItemsListingView('oa_listing_view.html', data);
+                // Add test data
+                view.oraData = testData;
+            });
 
-            view.renderGrids(ora2responses);
+            it('data context get parse correctly', () => expect(view.data).toEqual(data));
 
-            var items = $('.staff-esg-link');
-            var itemsCount = Object.keys(oraCourseItems).length;
-            // only 2 column should be esg-link. It is for staff and additional response column
-            expect(items.length / itemsCount).toEqual(2);
+            it('only 2 column should be esg-link. It is for staff and additional response column.', () => {
+                view.renderGrids(ora2responses);
+                var items = $('.staff-esg-link');
+                var itemsCount = Object.keys(oraCourseItems).length;
+                expect(items.length / itemsCount).toEqual(2);
 
-            for (let i = 0; i < items.length; i++) {
-                let item = items[0];
-                expect(item.href).toContain(data.CONTEXT.ORA_GRADING_MICROFRONTEND_URL);
-            }
+                for (let i = 0; i < items.length; i++) {
```

Comment:
this loop looks like it's not doing anything?

Reply:
Just to check if the url get sub into the column properly. I could just check one of them, however there isn't much different in my opinion.